app/org/sagebionetworks/bridge/s3/S3Helper.java

```diff
@@ -0,0 +1,46 @@
+package org.sagebionetworks.bridge.s3;
+
+import javax.annotation.Nonnull;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.S3Object;
+import com.google.common.base.Charsets;
+import com.google.common.io.CharStreams;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/** Helper class that simplifies reading S3 files. */
+@Component
+public class S3Helper {
+    private AmazonS3Client s3Client;
+
+    /** S3 Client. This is configured by Spring. */
+    @Autowired
+    public void setS3Client(AmazonS3Client s3Client) {
```

Comment:
We have multiple beans of AmazonS3Client.  It's the "s3CmsClient" that should be injected here.  The client has only permissions to the bucket holding the key pairs and it uses different IAM credentials for different stacks. However, @Autowired AFAIK is injection by type.  It is not clear which one is injected.  I'd suggest using @Resource to inject here.